src/components/views/project/projectDonateCard/GIVbackToast.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import {
+	brandColors,
+	Caption,
+	IconGIVBack,
+	IconHelp,
+	Subline,
+} from '@giveth/ui-design-system';
+import styled from 'styled-components';
+import { Flex, FlexCenter } from '@/components/styled-components/Flex';
+import useDetectDevice from '@/hooks/useDetectDevice';
+
+const GIVbackToast = () => {
+	const [isPopupOpen, setIsPopupOpen] = useState(false);
+	const { isMobile } = useDetectDevice();
+	const togglePopup = () => setIsPopupOpen(!isPopupOpen);
```

Comment:
Can we close the toast when user clicks outside just like all modals currently work? I believe it's better UX